R/conditions.R

```diff
@@ -130,6 +130,40 @@ captureStackTraces <- function(expr) {
 #' @include globals.R
 .globals$deepStack <- NULL
 
+getCallStackDigest <- function(callStack, warn = FALSE) {
+  dg <- attr(callStack, "digest", exact = TRUE)
+  if (!is.null(dg)) {
+    return(dg)
+  }
+
+  if (isTRUE(warn)) {
+    warning("Call stack doesn't have a cached digest; expensively computing one now")
+  }
+
+  digest::digest(getCallNames(callStack), algo = "md5")
```

Comment:
FWIW, it seems `rlang::hash()` could handle this and would be faster?

Reply:
Oh, TIL!

Reply:
It's much faster, thanks